hex_assignments/src/assignment.rs

```diff
@@ -75,36 +76,40 @@ impl HexAssignments {
             footfall,
             landtype,
             urbanized,
+            service_provider_override,
         } = self;
 
         use Assignment::*;
-        match (footfall, landtype, urbanized) {
+        match (footfall, landtype, urbanized, service_provider_override) {
+            // service provider override hex
+            // Overrides other dataset assignments if set
+            (_, _, _, A) => dec!(1.00),
```

Comment:
Do we need to handle (_, _, _, B) somehow?

Reply:
For service provider overrides a hex will only have one of two values ( A or C ).  A means the hex has an override, C means no override.
As such there is no need to handle B explicitly, if the value if not A then we ignore that assignment